waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManager.java

```diff
@@ -196,7 +196,7 @@ void open(HiveUgiArgs ugiArgs) {
       // Wait before launching the next round of connection retries.
       if (!isConnected && (retryDelaySeconds > 0) && ((attempt + 1) < retries)) {
         try {
-          LOG.info("Waiting " + retryDelaySeconds + " seconds before next connection attempt.");
+          LOG.debug("Waiting " + retryDelaySeconds + " seconds before next connection attempt.");
```

Comment:
does this log not help with detecting slow metastore issues ?

Reply:
I don't think so, we're already printing an info on line 124. This just adds to clutter imho.